src/Decorator/EchoDecorator.php

```diff
@@ -8,22 +8,13 @@ class EchoDecorator extends SafeEchoDecorator
 {
     /**
      * Sets the wrapped hiddenString for safe output.
-     *
-     * @param string $openString
-     * @param string $hiddenString
-     * @param mixed $data
-     *
-     * @return string
      */
     public function wrap(string $openString, string $hiddenString, $data = null): string
     {
         return $openString;
     }
 
     /**
-     * @param string $string
-     * @param string $hiddenChar
-     *
      * @return string
      */
     public function hide(string $string, string $hiddenChar = '*')
```

Comment:
MIssing `:string` after declaration. docblock is not required here anymore also.

Reply:
This is simply running php-cs-fixer. I am not making any changes to the methods' signatures. Doing so requires investigating whether or not it returns a nullable type. Most docblocks are inaccurate.